src/Microsoft.AspNetCore.Mvc.RazorPages/DependencyInjection/RazorPagesOptionsExtensions.cs

```diff
@@ -43,7 +43,7 @@ public static RazorPagesOptions ConfigureFilter(this RazorPagesOptions options,
         /// <param name="path">The path of the Razor Page.</param>
         /// <param name="policy">The authorization policy.</param>
         /// <returns>The <see cref="RazorPagesOptions"/>.</returns>
-        public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path, string policy)
+        public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path, string policy = "")
```

Comment:
We generally don't use optional parameters in public APIs. Could you create another overload?